src/utils/jwtutils.js

```diff
@@ -10,6 +10,7 @@ const generateToken=(user)=>{
       return jwt.sign(payload,secretkey,{expiresIn:"1h"})
 }
 
+
 module.exports={
```

Comment:
I recommend a space between the equality operator in the assignment statement unless it is passed as a parameter.